Room/app/src/main/java/com/example/room/data/models/types/MBTI.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.room.data.models.types
+
+enum class MBTI {
+    ISTJ,ISTP,ISFJ,ISFP,INTJ,INTP,INFJ,INFP,ESTJ,ESTP,ESFJ,ESFP,ENTJ,ENTP,ENFJ,ENFP
```

Comment:
```suggestion
    ISTJ, ISTP, ISFJ, ISFP, INTJ, INTP, INFJ, INFP, ESTJ, ESTP, ESFJ, ESFP, ENTJ, ENTP, ENFJ, ENFP
```
Reformat Code를 생활화합시다^^

Reply:
넵 명시하겠슴다!